plugins/woocommerce-admin/client/products/sections/product-details-section.tsx

```diff
@@ -49,9 +49,16 @@ export const ProductDetailsSection: React.FC = () => {
 		}
 	);
 
-	function doesNameHaveError(): boolean {
+	const hasNameError = () => {
```

Comment:
This name change seems unrelated to this PR.

Reply:
No it's not.  This is a general clean-up to match our naming conventions.  Do you think this warrants a separate PR?

Reply:
I ultimately don't think that this file should need to be modified to implement the SKU (re: my comment about `onBlur` below). So, if that is the case, then changing this function name in the context of this PR is unrelated. 

However, if this file needs to be modified, since this function is local to the file, I think it seems like a safe clean-up to make in the context of this PR, though still unnecessary and it does add a little more that should be tested/reviewed in the context of this PR (to make sure that name error handling still works, but honestly that is largely a compilation issue since we are in TS here). So, I'm cool with it!